feature_engine/creation/math_features.py

```diff
@@ -202,14 +265,117 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
         X_new: Pandas dataframe, shape = [n_samples, n_features + n_operations]
             The input dataframe plus the new variables.
         """
+
+        def np_transform(np_df, new_variable_names, np_variables, np_functions):
+            np_result_df = pd.DataFrame()
+            for np_function_idx, np_function in enumerate(np_functions):
+                if callable(np_function):
+                    np_function_name = np_function.__name__
+                else:
+                    np_function_name = np_function
+
+                if np_function_name in ("sum"):
+                    result = np.nansum(
+                        np_df[np_variables],
+                        axis=1,
+                    )
+                    np_result_df[new_variable_names[np_function_idx]] = pd.Series(
+                        result
+                    )
+                    continue
```

Comment:
why do we need continue here? so that it reads the following elif? should we not just use if then?